src/components/FlawAffects.vue

```diff
@@ -424,10 +426,49 @@ function selectAffects(event: Event) {
 }
 
 // Affects Fields
-function affectCvss3Vector(affect: ZodAffectType) {
-  affect.cvss_scores.find(({ issuer, cvss_version }) => issuer === 'RH' && cvss_version === 'V3')
-    ?.vector
-    || null;
+function affectCvss(affect: ZodAffectType) {
+  return affect.cvss_scores.find(({ issuer, cvss_version }) => issuer === 'RH' && cvss_version === 'V3');
+}
+
+function affectCvssDisplay(affect: ZodAffectType) {
+  const cvssScore = affectCvss(affect)?.score;
+  const cvssVector = useAffectCvss3Vector(affect).value;
+  if (cvssScore && cvssVector) {
+    return `${affectCvss(affect)?.score || ''} ${useAffectCvss3Vector(affect).value}`;
+  } else {
+    return useAffectCvss3Vector(affect).value;
+  }
+}
+
+function useAffectCvss3Vector(affect: ZodAffectType) {
+  return computed({
```

Comment:
This computed is doing nothing, each time the `useAffectCvss3Vector` is called a new instance is returned, and because is called inside a `v-for` each time `paginatedAffects` is updated all the affects will trigger the update